resize-observer-1/Overview.bs

```diff
@@ -231,6 +232,7 @@ interface ResizeObserverEntry {
     readonly attribute DOMRectReadOnly contentRect;
     readonly attribute ResizeObserverSize borderBoxSize;
     readonly attribute ResizeObserverSize contentBoxSize;
+    readonly attribute ResizeObserverSize devicePixelBorderBoxSize;
```

Comment:
In the discussion we decided the name should be about the content box, not the border box (though we did not decide on a particular name)

Reply:
Done.